lib/elixir/lib/file.ex

```diff
@@ -10,6 +10,12 @@ defmodule File do
   via `cp/3` and remove files and directories recursively
   via `rm_rf/1`.
 
+  Paths given to functions in this module can be either relative to the
+  current working directory (as returned by `File.cwd/0`), or absolute
+  paths. Shell conventions like `~` are not expanded automatically.
+  To use paths like `~/Downloads`, you can use `Path.expand/1-2` to
```

Comment:
Small question: does `Path.expand/1-2` get linked to the right function when generating documentation? If not, we can do "you can use `Path.expand/1` or `Path.expand/2` ...".